dev/tasks/info.clj

```diff
@@ -23,10 +23,10 @@
      (provided (get-in deps [:aliases :plk :extra-deps])))))
 
 (def options
-  {:lib           'djblue/portal
+  {:lib           'griffin/portal
```

Comment:
This PR looks awesome! I'm curious if you intend to maintain this fork long term? If so, is it due to ui extensibility mechanism in portal not being sufficient?

I was considering adding extensibility via shadow-cljs similar to [this](https://cljdoc.org/d/org.mentat/clerk-utils/0.4.1/api/mentat.clerk-utils.build.shadow#watch!) but wasn't sure who would be interested.

Reply:
Hello! First of all many thanks for portal :)

> I'm curious if you intend to maintain this fork long term?

No. In its current iteration, this code is useful for viewing our internal graphs and lacks a deeper integration with `portal` paths. Ideally, we would contribute this back to `portal` as a generic graph viewer. 

> If so, is it due to ui extensibility mechanism in portal not being sufficient?

Since we need additional npm packages to achieve this (like https://reactflow.dev/ ) a fork was the easiest route. If there was an extensibility mechanism I would have used it. However, if you are interested in having a reactflow-based graph viewer built-in `portal` this could be an option as well.